main.py

```diff
@@ -1,10 +1,13 @@
 from getter import Getter
 
+
 def main():
-    g = Getter()
-    user_input = input("Enter the best buy URL: \n")
-    g.getProductData()
-    
+    getter = Getter()
+    user_input = input("Enter the best buy URL: ")
+    if "bestbuy" in user_input.split() and "skuId=" in user_input.split():
```

Comment:
Added a validator in the main function so that the given url is verified as a valid url link. Used the contains function to check for strings 'bestbuy', and 'skuId'. Once all three return True, the function will continue as normal.